src/manage/approveAccessRequest.test.ts

```diff
@@ -19,10 +19,11 @@
 // SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 //
 
-/* eslint-disable no-shadow */
-import { mockSolidDatasetFrom } from "@inrupt/solid-client";
 import { jest, it, describe, expect } from "@jest/globals";
+
```

Comment:
Unecessary line break

Reply:
Was actually deliberate, as it's kinda a "good to do" to have the `@jest/globals` line be the first line in the file, and also separate from dependencies used for testing